openfisca_core/formulas.py

```diff
@@ -663,6 +618,61 @@ def get_neutralized_column(column):
         )
 
 
+def extract_formula_date_from_name(attribute_name):
+    """
+    Return a day date extracted from attribute name when it is a formula name, None otherwise.
+    Valid dated name formats are : 'formula', 'formula_YYYY', 'formula_YYYY_MM' and 'formula_YYYY_MM_DD' where YYYY, MM and DD are a year, month and day.
+
+    Default year is '0001'. Default month and day are '01'.
+    """
+    formula_name_prefix = 'formula'
+    if not attribute_name.startswith(formula_name_prefix):
+        # Current attribute isn't a formula
+        return None
+
+    formula_name_separator = '_'
```

Comment:
Why not declare all these constants at the same place than `formula_name_prefix`?